src/muya/lib/contentState/clickCtrl.js

```diff
@@ -12,6 +12,9 @@ const clickCtrl = ContentState => {
       const lastBlock = this.getLastBlock()
       const archor = this.findOutMostBlock(lastBlock)
       const archorParagraph = document.querySelector(`#${archor.key}`)
+      if (archorParagraph === null) {
```

Comment:
Why add these codes?

Reply:
I added this due to a TypeError that can be raised (pointed out in #2758) when `document.querySelector` returns null.

Reply:
I could change it to just `if (!archorParagraph) {`